network/src/peer_store/sqlite/peer_store.rs

```diff
@@ -146,44 +144,59 @@ impl SqlitePeerStore {
         }
 
         let tx = self.conn.transaction().expect("db tx");
-        db::PeerInfo::delete(&tx, candidate_peer.id).expect("delete peer error");
-        db::PeerAddr::delete_by_peer_id(&tx, candidate_peer.id)
+        db::PeerInfoDB::delete(&tx, &candidate_peer.peer_id).expect("delete peer error");
+        db::PeerAddrDB::delete_by_peer_id(&tx, &candidate_peer.peer_id)
             .expect("delete peer by peer_id error");
         tx.commit().expect("delete peer error");
         Ok(())
     }
 
-    fn fetch_peer_info(&self, peer_id: &PeerId) -> db::PeerInfo {
+    /// check and try delete peer_addr if peer_addr reach limit
+    fn check_peer_addrs_limit(&mut self, peer_id: &PeerId) -> Result<(), ()> {
```

Comment:
I don't understand why this function returns a `Result`, it doesn't return any `Err` in fn body.